projects/igniteui-angular/src/lib/list/list.component.spec.ts

```diff
@@ -445,6 +446,42 @@ describe('List', () => {
         });
     }, 5000);
 
+    it('check the panLeftTemplate is visible when left-panning a list item.', () => {
```

Comment:
Add a test for panning a header item.
Add tests for verifying the templates are not visible after releasing the item.

Reply:
More tests were added.